packages/react-native-codegen/src/parsers/error-utils.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+('use strict');
+
+import type {ParserType} from './errors';
+
+const {IncorrectModuleRegistryCallArityParserError} = require('./errors');
+
+function throwIfIncorrectModuleRegistryCallArityParserError(
+  nativeModuleName: string,
+  flowCallExpression: $FlowFixMe,
+  methodName: string,
+  incorrectArity: number,
```

Comment:
what about calling this `numberOfCallExpressionArgs`? I think that it conveys more semantic that just `incorrectArity`. Also, this is a parameter: it does not know there it is correct or not. It doesn't feel right to state `incorrect` in the parameter name. What do you think?